aten/src/ATen/native/quantized/cpu/qnnpack/src/conv-prepack.cc

```diff
@@ -142,9 +142,11 @@ PrePackConvWeights::PrePackConvWeights(
             packed_group_weights_size * groups);
         assert("QNNPACK Runtime Error.");
       }
+      // Setting packed weights to first channel's zero point.
+      // But this will be overwritten by weight packing.
```

Comment:
So this is needed in this PR because some kernels explicitly rely on this. Otherwise you get wrong result. The comment is referring to the fact that later PRs will overwrite this. Let me fix this comment. This is not yet removed in the later PRs, I realize. So I will fix this in the appropriate PR.